schemas/log.yml

```diff
@@ -19,22 +19,5 @@
     - name: offset
       type: long
       description: >
-        Offset of the beginning of the log event.
+        Offset from the beginning of the log event.
       example: 12
-    - name: message
```

Comment:
Not sure why this one is removed but it shouldn't be. Can you revert that part? Probably happened during rebasing.